python/services/notify_service.py

```diff
@@ -27,18 +22,26 @@ def send_first_email(self, email_address: str, login_date: str):
 
     def send_reminder_email(self, email_address: str, login_date: str):
         personalisation = {"login_date": login_date}
+
+        template_id = ""
+        if self.organisation_name == MINISTRY_OF_JUSTICE:
+            template_id = "7405b6f8-9355-4572-8b8c-c73bc8cdee3c"
```

Comment:
Be interested to know why we have two separate templates for the two orgs? 🤔 But maybe that's question for standup 😁

Reply:
Org or dept owner requested a different template to the one we use

Reply:
https://mojdt.slack.com/archives/CPVD6398C/p1674833375492059